<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgba(60.5625, 60.5625, 73.5675, 1)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1251</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="top_bar">
     <item>
      <widget class="QPushButton" name="learn_btn">
       <property name="minimumSize">
        <size>
         <width>186</width>
         <height>50</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: rgb(254, 190, 12);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    font: bold 14px;
    min-width: 10em;
    padding: 6px;
    color: rgba(60.5625, 60.5625, 73.5675, 1);
}
QPushButton:pressed {
    background-color: rgb(242, 178, 0);
    border-style: inset;
}</string>
       </property>
       <property name="text">
        <string>Learn by Demonstration</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="track_btn">
       <property name="minimumSize">
        <size>
         <width>186</width>
         <height>50</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: rgb(254, 190, 12);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    font: bold 14px;
    min-width: 10em;
    padding: 6px;
	color:  rgba(60.5625, 60.5625, 73.5675, 1);
}
QPushButton:pressed {
    background-color: rgb(242, 178, 0);
    border-style: inset;
}</string>
       </property>
       <property name="text">
        <string>Track object</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="phto_btn">
       <property name="minimumSize">
        <size>
         <width>186</width>
         <height>50</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: rgb(254, 190, 12);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    font: bold 14px;
    min-width: 10em;
    padding: 6px;
	color:  rgba(60.5625, 60.5625, 73.5675, 1);
}
QPushButton:pressed {
    background-color: rgb(242, 178, 0);
    border-style: inset;
}</string>
       </property>
       <property name="text">
        <string>Take Photo</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="halt_btn">
       <property name="minimumSize">
        <size>
         <width>186</width>
         <height>50</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: red;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    font: bold 14px;
    min-width: 10em;
    padding: 6px;
	color:  rgba(60.5625, 60.5625, 73.5675, 1);
}
QPushButton:pressed {
    background-color: rgb(224, 0, 0);
    border-style: inset;
}</string>
       </property>
       <property name="text">
        <string>STOP</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>311</width>
      <height>491</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="axes">
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Axis 3: </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Axis 2:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Axis 5: </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Base Rotation: </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Axis 4: </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Gripper: </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSlider" name="base_r">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSlider" name="a2_r">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSlider" name="a3_r">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSlider" name="a4_r">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QSlider" name="a5_r">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QSlider" name="grip_r">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>100</y>
      <width>160</width>
      <height>491</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="axes_readout">
     <item>
      <widget class="QLabel" name="a1_d">
       <property name="text">
        <string>Axis 1: </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="a2_d">
       <property name="text">
        <string>Axis 2:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="a3_d">
       <property name="text">
        <string>Axis 3:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="a4_d">
       <property name="text">
        <string>Axis 4:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="a5_d">
       <property name="text">
        <string>Axis 5:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="a6_d">
       <property name="text">
        <string>Axis 6:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>110</y>
      <width>120</width>
      <height>80</height>
     </rect>
    </property>
    <widget class="QWidget" name="viewfinder" native="true">
     <property name="geometry">
      <rect>
       <x>-30</x>
       <y>-10</y>
       <width>761</width>
       <height>491</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:black;</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>600</y>
      <width>1251</width>
      <height>112</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="bottom_bar">
       <item>
        <widget class="QPushButton" name="prev">
         <property name="minimumSize">
          <size>
           <width>186</width>
           <height>50</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    background-color: rgb(254, 190, 12);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    font: bold 14px;
    min-width: 10em;
    padding: 6px;
    color: rgba(60.5625, 60.5625, 73.5675, 1);
}
QPushButton:pressed {
    background-color: rgb(242, 178, 0);
    border-style: inset;
}</string>
         </property>
         <property name="text">
          <string>&lt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="next">
         <property name="minimumSize">
          <size>
           <width>186</width>
           <height>50</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    background-color: rgb(254, 190, 12);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    font: bold 14px;
    min-width: 10em;
    padding: 6px;
	color:  rgba(60.5625, 60.5625, 73.5675, 1);
}
QPushButton:pressed {
    background-color: rgb(242, 178, 0);
    border-style: inset;
}</string>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="execute">
         <property name="minimumSize">
          <size>
           <width>186</width>
           <height>50</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    background-color: rgb(254, 190, 12);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    font: bold 14px;
    min-width: 10em;
    padding: 6px;
	color:  rgba(60.5625, 60.5625, 73.5675, 1);
}
QPushButton:pressed {
    background-color: rgb(242, 178, 0);
    border-style: inset;
}</string>
         </property>
         <property name="text">
          <string>&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="bottom_bar_2">
       <item>
        <widget class="QPushButton" name="follow_red">
         <property name="minimumSize">
          <size>
           <width>186</width>
           <height>50</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    background-color: rgb(254, 190, 12);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    font: bold 14px;
    min-width: 10em;
    padding: 6px;
    color: rgba(60.5625, 60.5625, 73.5675, 1);
}
QPushButton:pressed {
    background-color: rgb(242, 178, 0);
    border-style: inset;
}</string>
         </property>
         <property name="text">
          <string>Follow Red</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="follow_green">
         <property name="minimumSize">
          <size>
           <width>186</width>
           <height>50</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    background-color: rgb(254, 190, 12);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    font: bold 14px;
    min-width: 10em;
    padding: 6px;
	color:  rgba(60.5625, 60.5625, 73.5675, 1);
}
QPushButton:pressed {
    background-color: rgb(242, 178, 0);
    border-style: inset;
}</string>
         </property>
         <property name="text">
          <string>Follow Green</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="follow_blue">
         <property name="minimumSize">
          <size>
           <width>186</width>
           <height>50</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    background-color: rgb(254, 190, 12);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    font: bold 14px;
    min-width: 10em;
    padding: 6px;
	color:  rgba(60.5625, 60.5625, 73.5675, 1);
}
QPushButton:pressed {
    background-color: rgb(242, 178, 0);
    border-style: inset;
}</string>
         </property>
         <property name="text">
          <string>Follow Blue</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="follow_stop">
         <property name="minimumSize">
          <size>
           <width>186</width>
           <height>50</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    background-color: rgb(254, 190, 12);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    font: bold 14px;
    min-width: 10em;
    padding: 6px;
	color:  rgba(60.5625, 60.5625, 73.5675, 1);
}
QPushButton:pressed {
    background-color: rgb(242, 178, 0);
    border-style: inset;
}</string>
         </property>
         <property name="text">
          <string>Stop Following</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>learn_btn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>164</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel">
     <x>639</x>
     <y>359</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
